### PR TITLE
Fix the number of parameters passed to UnexportableArtifactException

### LINKED ISSUE #3313: `UnexportableArtifactException` does not expect any arguments during the initialization
`UnexportableArtifactException.__init__() takes 1 positional argument but 2 were given)'`

https://github.com/pulp/pulpcore/blob/351bd8653820195ffd3b16c5fbab0f14fd0061fe/pulpcore/app/tasks/export.py#L41-L59

### PR DESCRIPTION
**Pull request by lubosmj**
closes #3313

### CODE REVIEW

**Review by mdellweg — CHANGES_REQUESTED** [review]
I assume this is fixing a bug. So we need an issue for all the downstream trackers.

File: CHANGES/3313.bugfix [hunk 1]
```diff
@@ -0,0 +1 @@
+Fixed an error raised when initializing ``UnexportableArtifactException``.
```
Comment (mdellweg) [inline CHANGES/3313.bugfix:1]:
```suggestion
Fixed an error when raising ``UnexportableArtifactException``.
```

**Review by mdellweg — APPROVED** [review]
A suggestion for the Changelog wording.
Fine for me anyway.

**Review by ggainey — COMMENTED** [review]
lg2m

**Review by ggainey — APPROVED** [review]

**Comment by ggainey** [comment]
Adding backports to support released downstream products

**Pull request merged**